immersion/kubeflow_pipelines/walkthrough/solutions/lab-01_vertex.ipynb

```diff
@@ -0,0 +1,880 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    from kfp.components import func_to_container_op
</pre><p>for this to work, we need <code>pip install kfp</code>, could we maybe have the correct pip install commented in a cell above (together with a reminder to restart kernel), so people know how to install, but avoid unnecessary pip runs</p><p> </p><p>such as</p><pre class="ql-syntax" spellcheck="false"># !pip3 install -U kfp --user
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>